src/bundle/Resources/public/scss/_buttons.scss

```diff
@@ -25,7 +25,7 @@
     }
 }
 
-.ez-btn--wide {
+.ez-btn--wide.ez-btn--wide.ez-btn--wide {
```

Comment:
This maybe is clever but not really understandable at first look and someone in the future may remove this because will take this as a mistake.

I would go with regular `.ez-btn.ez-btn--wide`

Reply:
We need 3 classes, because `.ez-context-menu .btn` is defined below `.ez-btn--wide` in generated CSS. Do we want `.ez-btn.ez-btn.ez-btn--wide`?

Reply:
Let's just move it below the `.ez-context-menu .btn` and use the selector for "BC":
```
.ez-btn--wide,
.ez-btn.ez-btn--wide
```